daml-lf/spec/daml-lf-1.rst

```diff
@@ -475,7 +475,7 @@ DAML-LF as a "not an Ident" notation, so should not be considered for
 future addition to allowed identifier characters.
 
 In the following, we will use identifiers to represent *built-in
-functions*, term and type *variable names*, record and tuple *field
+functions*, term and type *variable names*, record and struct *field
```

Comment:
We should explain somewhere that "record" means nominal record and "struct" means structural records.

Reply:
done